pkg/gatherers/clusterconfig/service_accounts.go

```diff
@@ -96,32 +96,35 @@ func gatherServiceAccounts(ctx context.Context, coreClient corev1client.CoreV1In
 
 // ServiceAccountsMarshaller implements serialization of Service Accounts
 type ServiceAccountsMarshaller struct {
-	sa                   []corev1.ServiceAccount
+	serviceAccounts      []corev1.ServiceAccount
 	totalServiceAccounts int
 }
 
 // Marshal implements serialization of ServiceAccount
 func (a ServiceAccountsMarshaller) Marshal() ([]byte, error) {
 	// Creates map for marshal
-	sr := map[string]interface{}{}
-	st := map[string]interface{}{}
-	st["TOTAL_COUNT"] = a.totalServiceAccounts
-	sr["serviceAccounts"] = st
-	nss := map[string]interface{}{}
-	st["namespaces"] = nss
-	for i := range a.sa {
-		var ns map[string]interface{}
-		var ok bool
-		if _, ok = nss[a.sa[i].Namespace]; !ok {
-			ns = map[string]interface{}{}
-			nss[a.sa[i].Namespace] = ns
-		} else {
-			ns = nss[a.sa[i].Namespace].(map[string]interface{})
+	serviceAccounts := map[string]any{}
```

Comment:
This line seems useless. Can't you just do 106 and 108 in the same line?

Reply:
I have tried that before and it doesn't work unfortunately :/

Reply:
something like:
`serviceAccounts := map[string]any{
		"serviceAccounts" : map[string]any{
			"TOTAL_COUNT": a.totalServiceAccounts,
			"namespaces":  namespaces,
		},
	}`
would work but I think it's better the way it is now

Reply:
You can't. Map has to be initialized in Go.